main.go

```diff
@@ -30,14 +31,22 @@ func EndGame() {
 }
 
 func main() {
+	var isFullscreen = flag.Bool("fullscreen", false, "Play fullscreen!")
```

Comment:
Change to `isFullScreen := flag.Bool(...)` to be more consistent with other variables.

Reply:
Fixed!